src/ecs/fargate/README.md

```diff
@@ -170,12 +170,13 @@ For more general information, reference the [Datadog ECS Fargate Docs](https://d
 
 ### DogstatsdFeatureConfig
 
-| Property                   | Type          | Description                                                                      |
-| -------------------------- | ------------- | -------------------------------------------------------------------------------- |
-| `isEnabled`                | `boolean`     | Enables DogStatsD.                                                               |
-| `isOriginDetectionEnabled` | `boolean`     | Enables DogStatsD origin detection.                                              |
-| `dogstatsdCardinality`     | `Cardinality` | Controls the cardinality of custom DogStatsD metrics.                            |
-| `isSocketEnabled`          | `boolean`     | Enables DogStatsD traffic over Unix Domain Socket. Falls back to UDP when false. |
+| Property                     | Type          | Description                                                                                                                                                                                      |
+| ---------------------------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `isEnabled`                  | `boolean`     | Enables DogStatsD.                                                                                                                                                                               |
+| `isOriginDetectionEnabled`   | `boolean`     | Enables DogStatsD origin detection.                                                                                                                                                              |
+| `dogstatsdCardinality`       | `Cardinality` | Controls the cardinality of custom DogStatsD metrics.                                                                                                                                            |
+| `isSocketEnabled`            | `boolean`     | Enables DogStatsD traffic over Unix Domain Socket. Falls back to UDP when false.                                                                                                                 |
+| `traceInferredProxyServices` | `boolean`     | Enables inferred spans for proxy services like AWS API Gateway. When enabled, the tracer will create spans for proxy services by using headers passed from the proxy service to the application. |
```

Comment:
is it the right place for this config option? 
I might wrong, but it seems to be more related to `APMFeatureConfig` than `DogstatsdFeatureConfig`.

Reply:
Ah, thank you, good catch!